packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -40,6 +61,167 @@ abstract class BindingBase {
     assert(() { _debugInitialized = true; return true; });
   }
 
+  /// Called, in checked mode, to register service extensions.
```

Comment:
The driver extension needs to be able to run in unchecked mode because it is used to test performance.